bedrock/redirects/redirects.py

```diff
@@ -2005,6 +2009,59 @@
+    redirect(r'^thunderbird/early_releases/downloads/?$', 'https://www.thunderbird.net/thunderbird/beta/all/'),
+    # bug 1124042
+    redirect(r'^thunderbird/features/email_providers\.html$', 'https://www.thunderbird.net/email-providers/'),
+    # bug 1081917, 1029829, 1029838
+    redirect(r'^thunderbird/releases/(?P<version>0\.\d)\.html$',
+             'http://website-archive.mozilla.org/www.mozilla.org/thunderbird_releasenotes'
+             '/en-US/thunderbird/releases/{version}.html'),
+    # should catch everything 1.* to 29.*
+    redirect(r'^thunderbird/(?P<version>(?:\d|[12]\d)\.[^/]+)/'
+             '(?P<page>firstrun|releasenotes|start|system-requirements|whatsnew)/$',
+             'http://website-archive.mozilla.org/www.mozilla.org/thunderbird_releasenotes'
+             '/en-US/thunderbird/{version}/{page}/'),
+    # bug 1209720
+    redirect(r'^thunderbird/releasenotes/?$', 'https://www.thunderbird.net/thunderbird/releases/'),
+    # bug 1211007
+    redirect(r'^thunderbird/download/?', 'https://www.thunderbird.net/'),
+    # bug 1133266
+    redirect(r'^thunderbird/legal/privacy/?$', 'privacy.notices.thunderbird'),
+    redirect(r'^thunderbird/about/privacy-policy/?$',
+             'privacy.archive.thunderbird-2010-06'),
+    # bug 1196578
+    redirect(r'^thunderbird/about/legal/eula/?$', 'legal.eula'),
+    redirect(r'^thunderbird/about/legal/eula/thunderbird2',
+             'legal.eula.thunderbird-2-eula'),
+    redirect(r'^thunderbird/about/legal/eula/thunderbird',
+             'legal.eula.thunderbird-1.5-eula'),
+    # bug 1204579
+    redirect(r'^thunderbird/2.0.0.0/eula/?$', 'legal.eula.thunderbird-2-eula'),
+    redirect(r'^thunderbird/about/legal/?$', 'legal.terms.mozilla'),
+    redirect(r'^thunderbird/about(/mission)?/?$',
+             'https://wiki.mozilla.org/Thunderbird'),
+    redirect(r'^thunderbird/(about/(careers|contact|get-involved)|community)/?$',
+             'https://wiki.mozilla.org/Thunderbird#Contributing'),
+    redirect(r'^thunderbird/(?P<version>\d\.\d(?:a|b|rc)\d|[6-9]\.0beta)/?$',
+             'http://website-archive.mozilla.org/www.mozilla.org/thunderbird'
+             '/thunderbird/{version}/'),
+    redirect(r'^thunderbird/about/(?P<page>board|press|staff)/',
+             'http://website-archive.mozilla.org/www.mozilla.org/thunderbird'
+             '/thunderbird/about/{page}/'),
+    # bug 1388914
+    redirect(r'^thunderbird/all/$', 'https://www.thunderbird.net/thunderbird/all/'),
+    redirect(r'^thunderbird/beta/(?P<path>.*)$', 'https://www.thunderbird.net/thunderbird/beta/{path}'),
+    redirect(r'^thunderbird/(?P<version>[^/]+)/releasenotes/$',
+             'https://www.thunderbird.net/thunderbird/{version}/releasenotes/'),
+    redirect(r'^thunderbird/(?P<version>[^/]+)/system-requirements/$',
+             'https://www.thunderbird.net/en-US/thunderbird/{version}/system-requirements/'),
```

Comment:
This is likely from an old test, but we can probably remove the hard coded `/en-US/` here and thunderbird.net can manage the locale redirection itself?